rpcs3/Emu/RSX/Program/GLSLSnippets/RSXProg/RSXROPEpilogue.glsl

```diff
@@ -0,0 +1,63 @@
+R"(
+
+#ifdef _DISABLE_EARLY_DISCARD
+	if (_fragment_discard)
+	{
+		discard;
+	}
+#endif

```

Comment:
if you disable early discard, why are you discarding here? shouldn't it be ifndef ?

Reply:
This is late discard. Modern glsl has a workaround by replacing discard with demote but I don't feel like wrangling another extension.